.github/workflows/columnq_cli_release.yml

```diff
@@ -3,10 +3,11 @@ name: columnq-cli release
 on:
   push:
     paths:
+      - '.github/workflows/columnq_cli_release.yml'
       - 'columnq/**'
       - 'columnq-cli/**'
       - 'Cargo.lock'
-    branches: [ main ]
+    branches: [ main, rustls ]
```

Comment:
i believe rustls was added for testing purpose?

Reply:
definitely :)